ocaml/xapi/nm.ml

```diff
@@ -462,18 +462,16 @@ let bring_pif_up ~__context ?(management_interface=false) (pif: API.ref_PIF) =
             let master = Db.Bond.get_master ~__context ~self:bond in
             Db.PIF.set_currently_attached ~__context ~self:master ~value:false
         end;
-
-        (* sync MTU *)
-        (try
-           let mtu = Int64.of_int (Net.Interface.get_mtu dbg ~name:bridge) in
-           Db.PIF.set_MTU ~__context ~self:pif ~value:mtu
-         with _ ->
-           debug "could not update MTU field on PIF %s" rc.API.pIF_uuid
-        );
-
         Xapi_mgmt_iface.on_dom0_networking_change ~__context
-      end
-    )
+      end;
+
+      (* sync MTU *)
+      try
+        let mtu = Int64.of_int (Net.Interface.get_mtu dbg ~name:bridge) in
+        Db.PIF.set_MTU ~__context ~self:pif ~value:mtu
+      with _ ->
+        debug "could not update MTU field on PIF %s" rc.API.pIF_uuid
```

Comment:
This line wasn't changed, just re-indented. But yes, let's fix it and make it a `warn` now that we are here.